app/models/piece.rb

```diff
@@ -18,6 +18,27 @@ def move!(x,y)
     end
   end
 
+  #Inital Capture Logic:
+  #Need game model to test id of game
+
+  #Set initial captured status to false
+  def default_captured_status
+    self.captured = false if self.captured = nil
+  end
```

Comment:
We will just set the default value in the database for the captured flag to false.  No need to set it manually for every piece.  This method is not needed.